sample_rock/rockcraft.yaml

```diff
@@ -11,37 +11,19 @@ platforms:
   amd64:
 
 parts:
-  flask-environment:
-    plugin: nil
-    overlay-script: |
-      mkdir $CRAFT_OVERLAY/etc
-      mkdir $CRAFT_OVERLAY/tmp
-      chmod 755 $CRAFT_OVERLAY/etc
-      chmod 777 $CRAFT_OVERLAY/tmp
-      groupadd -R $CRAFT_OVERLAY --gid 2000 flask
-      useradd -R $CRAFT_OVERLAY --system --gid 2000 --uid 2000 --home /srv/flask flask
   flask-dependencies:
-    stage-packages:
-      - python3-dev
-      - python3-gunicorn
-    build-packages:
-      - build-essential
-      - python3-pip
+    plugin: python
     source: .
-    plugin: nil
-    overlay-script: |
-      mkdir -p $CRAFT_OVERLAY/srv/flask/.local/lib/python3.10/site-packages
-      pip install -r $CRAFT_PART_SRC_WORK/sample-app/requirements.txt --target $CRAFT_OVERLAY/srv/flask/.local/lib/python3.10/site-packages
-      chown 2000:2000 -R $CRAFT_OVERLAY/srv/flask/
+    stage-packages:
+      - python3-venv
+    python-packages:
+      - gunicorn
```

Comment:
Could you also add flask in the systematically installed packages ?

Reply:
We don't leave the choice of the flask version to the end user ?

Reply:
nvm, I understand that the user can give its own OCI images

Reply:
See description of the PR on how we decided to proceed.